src/utils/fetch-config.ts

```diff
@@ -20,7 +20,8 @@ export async function fetchConfig (): Promise<void> {
   // fetch config from API
   // eg, http://localhost:8080/business/config/configuration.json
   // eg, https://dev.bcregistry.ca/business/config/configuration.json
-  const url = `${windowLocationOrigin}${processEnvBaseUrl}config/configuration.json`
+  let random = new Date().toISOString().substring(0, 10)
+  const url = `${windowLocationOrigin}${processEnvBaseUrl}config/configuration.json?${random}`
```

Comment:
Why do you need both this change (which should bust the cache, I think), and the change in `vue.config.js` (which should skip pre-caching) ? Why isn't one of these changes enough ?

Reply:
the pre-caching fixes the service worker cache.But the file gets cached in the disk. So it was getting served still from the cache.
We thought closing the browser and logout will invalidate the cache , but it didnt. Hence have to go for the fix.

Reply:
The ideal fix would be to serve this file from nginx with no-cache header .

Reply:
^^^ Yes, I agree.

I'm going to put this on my back-burner until it is decided how to proceed with this (re: Thor's planning).

Reply:
I am good with that..do you want to  tag/add it to the release ? Also incase if the planning/implementation doesn't happen before next release , it will be good to have this code in the next release as well.. But all leaving all this up to you.  :-)

Reply:
@thorwolpert Please let us know your plan for this moving forward. Thanks!